docs/components/DeprecationNotice.js

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import { Flex, Typography, ThemeProvider, lightTheme } from '@strapi/design-system';
+
+export const DeprecationNotice = ({ children, href }) => (
+  <ThemeProvider theme={lightTheme}>
+    <Flex padding={5} background="danger200" justifyContent="center" marginTop={4} marginBottom={4}>
+      <Typography fontSize={4} fontWeight="bold" variant="beta">
+        ⛔️
+        <strong>
```

Comment:
I think this is already bold from the Typography component?

Reply:
The font-size isn't inherited, so you end up having to do:

```jsx
      <Typography fontSize={4} as="p">
        ⛔️
        <Typography fontSize={4} as="strong" fontWeight="bold">
          {' This component has been deprecated. Please use '}
          <a style={{ color: 'inherit', textDecoration: 'underline' }} href={href}>
            {children}
          </a>
          {' instead. '}
        </Typography>
        ⛔️
      </Typography>
```

Which i don't think is better? Although i will add `as="p"` in, i'm surprised that isn't the default 👀